lib/directory-monitor-inotify.c

```diff
@@ -56,6 +56,10 @@ _get_event_type(struct inotify_event *event, gchar *filename)
     {
       return DIRECTORY_DELETED;
     }
+  else if (event->mask & IN_MODIFY)
```

Comment:
Similar question again: do we need to change inotify based monitoring?

Reply:
I see now inotify needs to be changed too from your explanation above. Resolving the comment.